walletunlocker/service.go

```diff
@@ -56,6 +56,7 @@ func New(authSvc *macaroons.Service, chainDir string,
 		UnlockPasswords: make(chan []byte, 1),
 		chainDir:        chainDir,
 		netParams:       params,
+		authSvc:         authSvc,
```

Comment:
Used for anything?

Reply:
`authSvc` is used in `InitWallet` and `UnlockWallet` to create/unlock the macaroon database. Looks like it was just forgot to be set.

Reply:
So this means the macaroon database was not really encrypted before? 🤔